### ISSUE TITLE
NTR: Methionine sulfoxide high in blood (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8530]
New term request Methionine sulfoxide high in blood (segal_030506182947):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000267
- label: Elevated circulating methionine sulfoxide concentration
- parents: [Abnormal circulating proteinogenic amino acid derivative concentration (HP:0033108)](https://hpo.jax.org/app/browse/term/HP:0033108)
- pmid: 
- def: The concentration of methionine sulfoxide in the blood circulation is above the upper limit of normal.
- comment: Methionine S-oxide is a derivative of methionine and is a biomarker of oxidative stress.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8530
- synonyms:
